### DISCUSSION TITLE
Need help for clipboard copy

### DISCUSSION THREAD

**Discussion by MateiVladFlorian** [MateiVladFlorian/Simple_calculator#5] [General]
How to copy text from clipboard into winforms project.
Can u help?! 😄

**Comment by MateiIonutEduard** [comment]
Hello @MateiVladFlorian! Wow, sure I can help you bro, in the **System.Windows.Forms** namespace, you have
a class named **Clipboard** that provides you some methods to manipulate multiple **MIME** data format such as
text, images etc. So, in your case, just insert the following lines of code:<br/>
```csharp
if (Clipboard.ContainsText())
{
     // Do whatever you need to do with clipboard text
     string clipboardText = Clipboard.GetText();
}
```

**Reply by MateiVladFlorian** [reply]
Wow, thanks, I think that's help 😉
